scripts/realtime.in

```diff
@@ -103,10 +103,13 @@ CheckConfig(){
         MODULES_UNLOAD="$MOD $MODULES_UNLOAD"
     done
     case $RTPREFIX in
-    *rtai*)
+    (*rtai*)
```

Comment:
This use of '(' is quite unusual in a case/esac block.  Why do you want it?

Reply:
Naive text editors (those that perform paren matching but don't fully understand language syntax) always want parens to match. It's just a stylistic thing and can be dropped here if it's controversial.